app/src/main/java/com/android/wardrobeManager/ExpandClosetActivity.java

```diff
@@ -1,25 +1,94 @@
 package com.android.wardrobeManager;
 
 import android.content.Intent;
+import android.graphics.PorterDuff;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
 import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.ArrayList;
 
 public class ExpandClosetActivity extends AppCompatActivity {
 
+    private ArrayList<ClothingItem> closetClothes = null;
+    private ClothingItem newClothingItem = null;
+    private String previousActivity = null;
+    private ImageView newClothingItemImage = null;
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_expand_closet);
+        closetClothes = getIntent().getParcelableArrayListExtra("closetClothes");
+        previousActivity = getIntent().getStringExtra("previousActivity");
+        newClothingItem = getIntent().getParcelableExtra("newClothingItem");
+        newClothingItemImage = findViewById(R.id.newClothingItemImage);
+
+        if (newClothingItem != null) {
+
+            String clothingType = newClothingItem.clothingTypeToString();
+
+            switch (clothingType) {
+                case "Shirt":
+                    newClothingItemImage.setImageResource(R.drawable.gray_shirt);
+                    break;
+                case "Shoes":
+                    newClothingItemImage.setImageResource(R.drawable.gray_shoes);
+                    break;
+                case "Shorts":
+                    newClothingItemImage.setImageResource(R.drawable.gray_shorts);
+                    break;
+            }
+            updateColor();
+        }
 
     }
 
+    public void updateColor() {
+
+        newClothingItemImage.setColorFilter(newClothingItem.getColor(), PorterDuff.Mode.OVERLAY);
+
+    }
+
+    public void addItemToCloset(View view) {
+        if (newClothingItem != null) {
+            closetClothes.add(newClothingItem);
+
+            ((TextView)view).setText("Add duplicate\nto closet");
+        }
+    }
```

Comment:
Get rid of this functionality for now. We can add it back in later, but I don't think there will be much use to this aside from like socks or underwear or something. We can worry about it in another PR